taurus/ingest/tests/test_material_run_example.py

```diff
@@ -35,7 +35,7 @@
             "knob_2_setting": "high",
             "temperature": 624,
             "density": "0.80 +- 0.12",
-            "kinematic viscosity": "1.0e3 St",
+            "kinematic viscosity": "0.1",
```

Comment:
Why remove viscosity units?  This would seem important to polymer and melt studies.

Reply:
Its not on @lkubie 's list.